moonraker/components/notifier.py

```diff
@@ -123,6 +163,16 @@ def __init__(self, config: ConfigHelper) -> None:
         self.apprise = apprise.Apprise()
         self.attach = config.gettemplate("attach", None)
         url_template = config.gettemplate("url")
+        self.layer_trigger = float(config.get("layer_trigger", 0))
+        self.minimum_layers = int(config.get("minimum_layers", 0))
+        if self.layer_trigger < 0 or self.layer_trigger > 1:
+            # layer_trigger out of range. Default to 0 and send logging info
+            logging.info(
+                f"Layer change trigger out of range for Notifier {self.name}. "
+                + f"Expected decimal between 0 and 1, found {self.layer_trigger}. "
+                + "Ignoring layer change trigger."
```

Comment:
Operators can be skipped, strings between brackets will still be concatenated:
```suggestion
                f"Layer change trigger out of range for Notifier {self.name}. "
                f"Expected decimal between 0 and 1, found {self.layer_trigger}. "
                "Ignoring layer change trigger."
```